task_three_and_four/src/test/java/com/epam/webdriver/aleksandrgostev/driver/DriverManager.java

```diff
@@ -14,13 +16,26 @@ private DriverManager() {
     }
 
     public static WebDriver getDriver() {
-        if (driver.get() == null) {
-            driver.set(new ChromeDriver());
-
+        if (null == driver.get()) {
+//            //How to run test with Param=kkk via Intellij IDEA?
+//            System.out.println(System.getProperty("kkk"));
+            switch (System.getProperty("browser")) {
+                case "firefox": {
+                    WebDriverManager.firefoxdriver().setup();
+                    driver.set(new FirefoxDriver());
+                }
+                case "edge": {
+                    WebDriverManager.edgedriver().setup();
+                    driver.set(new FirefoxDriver());
+                }
+                default: {
```

Comment:
Let's add "chrome" option for transparency, it would look like this:
case "chrome":
                default: {
                    WebDriverManager.chromedriver().setup();
                    driver.set(new ChromeDriver());
                }
i.e. whenever you set -Dbrowser to chrome or either not set it at all, chrome driver will be created